testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
+name=rabbitmq
+daemon=/usr/sbin/rabbitmq-server
+
+depend() {
+	need net localmount
+	after firewall
+}
+
+start() {
+	ebegin "Starting ${name}"
+	# We need to test if /var/run/rabbitmq exists, since scirpt will fail if it does not
+	[ ! -e /var/run/rabbitmq ] && \
+		mkdir /var/run/rabbitmq && \
+		chown rabbitmq:rabbitmq /var/run/rabbitmq
+
+    
+    
+	start-stop-daemon --start \
+		--env RABBITMQ_PID_FILE=$pidfile \
+		--pidfile $pidfile \
+		--exec /bin/sh -- -c "$daemon \
+		> /var/log/rabbitmq/startup_log \
+		2> /var/log/rabbitmq/startup_err 0<&- &"
```

Comment:
Their shell wrapper doesn't detach from foreground, so I think that's why their init script does the & business.  I do see that start-stop-daemon has a -b option that might make more sense than the , and your suggestion of --stdout and --stderr is a good one.